src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -181,12 +180,13 @@ TIP: If all the nodes of an AWS instance are removed, the instance will be termi
     -   *minRam:* The minimum required amount of RAM expressed in Mega Bytes for each AWS instance that needs to be created.
     
     -   *minCores:* The minimum required amount of virtual cores for each AWS instance that needs to be created.
-    
+
 +
 WARNING: If the combination between RAM and CORES does not match any existing AWS instance type, then the closest to the specified parameters will be selected.
-+
-    
 
+    -   *spotPrice:* Amazon EC2 Spot instances allow you to bid on spare Amazon EC2 computing capacity. Since Spot instances are often available at a discount compared to On-Demand pricing. More information available documentation on https://aws.amazon.com/ec2/spot/[AWS Spot]
```

Comment:
Can you add "The maximum price that you are willing to pay per hour per instance (your bid price). "